app/instance-initializers/features-setup.js

```diff
@@ -0,0 +1,17 @@
+import config from 'frontend-gelinkt-notuleren/config/environment';
+
+export function initialize(appInstance) {
+  const features = appInstance.lookup('service:features');
+  const featureConfig = config.featureFlags || {};
+  for (const key of Object.keys(featureConfig)) {
+    const val = featureConfig[key];
+    if (typeof val === 'string' || val instanceof String) {
```

Comment:
Do we need both checks? Is there any string that is not typeof string but instanceof String?

Reply:
ok, then good job for catching that :)